### PR TITLE
fix: localstorage for last market

### PR DESCRIPTION
**Pull request by KngZhi**
## Description

Fix last market in localstorage

## Related issue

issue #1057

### CODE REVIEW

**Review by platschi — DISMISSED** [review]
Tested PR deploy and works as expected :+1:

**Comment by LeifuChen** [comment]
@KngZhi good catch about the weird string. I think the root cause is here https://github.com/Kwenta/kwenta/blob/dev/sections/futures/Trade/MarketsDropdown.tsx#L155
After that, you may also change the code here https://github.com/Kwenta/kwenta/blob/dev/sections/shared/Layout/AppLayout/Header/Nav/Nav.tsx#L30

Thanks for your help.

File: sections/futures/FuturesMarketTabs/FuturesMarketTab.tsx [hunk 1]
```diff
@@ -63,7 +68,9 @@ const FuturesMarketsTable: FC<FuturesMarketsTableProps> = ({
 				data={data}
 				showPagination={true}
 				onTableRowClick={(row) => {
-					router.push(`/market/${row.original.asset}`);
+					const asset = `/market/${row.original.asset}`;
```
Comment (LeifuChen) [inline sections/futures/FuturesMarketTabs/FuturesMarketTab.tsx:71]: After you fixing the issue I mentioned above, we can remove this.

**Review by LeifuChen — CHANGES_REQUESTED** [review]

**Review by LeifuChen — APPROVED** [review]
@KngZhi Looks good to me. Thanks for your contributions.

**Comment by KngZhi** [comment]
Thanks for your guidance! :)

**Review by platschi — APPROVED** [review]
Thank you! Works as requested.

**Pull request merged**